unit_tests/test_ahb_csv_reader.py

```diff
@@ -179,6 +179,16 @@ def test_csv_file_reading_11042(self, datafiles):
         path_to_csv: Path = datafiles / "11042.csv"
         reader = FlatAhbCsvReader(file_path=path_to_csv)
         assert len(reader.rows) == 846
+        assert (
+            len(
+                [
+                    r
+                    for r in reader.rows
+                    if r.section_name == "Korrespondenzanschrift des Kunden des Messstellenbetreibers"
```

Comment:
![grafik](https://user-images.githubusercontent.com/23094997/166577022-ef859c65-e61c-4a5e-8f9e-c0b16c25d1d5.png)

die einzige mehrzeilige überschrift in 11042, die auch ein kreuzchen hat (nicht nur bei 11042, sondern *auch* bei 11042)